foundation-ui/navigation/EventTypes.js

```diff
@@ -0,0 +1,3 @@
+export default {
```

Comment:
I'll change it, but I'd even prefer having multiple named exports like 
```
export const NAVIGATION_CHANGE = 'NAVIGATION_CHANGE';
```
but then we have eslint rule prefer-default-export :trollface: